### PR TITLE
CT-2847 Apple Pay disappearing settings fix

### PR DESCRIPTION
**Pull request by RafalOzogJudopay**
It's not the final solution necessarily, but the best I have come up with as far.

The comment may be removed, I would like you to decide whether it's useful or unnecessary. In any case it explains how the fix works, so I don't repeat myself here.

### CODE REVIEW

File: Examples/ObjectiveCExampleApp/Podfile.lock [hunk 1]
```diff
@@ -121,4 +121,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 058be2228693a67feaa663db1c657ea71d485c39
 
-COCOAPODS: 1.14.3
+COCOAPODS: 1.13.0
```
Comment (stefan-tudor) [inline Examples/ObjectiveCExampleApp/Podfile.lock:124]: please revert this

Comment (stefan-tudor) [inline Examples/ObjectiveCExampleApp/Podfile.lock:124]: please revert this

**Review by stefan-tudor — APPROVED** [review]

File: Examples/ObjectiveCExampleApp/Podfile.lock (hunk 1 above)
Comment (RafalOzogJudopay) [reply Examples/ObjectiveCExampleApp/Podfile.lock]: done

Comment (RafalOzogJudopay) [reply Examples/ObjectiveCExampleApp/Podfile.lock]: done

**Review by eugenezhernakov — APPROVED** [review]

**Pull request merged**